whitepaper/wlroots.tex

```diff
@@ -0,0 +1,536 @@
+\documentclass{article}
+\usepackage{hyperref}
+\usepackage[nohyphen]{underscore}
+\usepackage[english]{babel}
+\newcommand{\code}[1]{\texttt{#1}}
+\begin{document}
+\title{Modular Wayland compositors with wlroots}
+\author{Drew DeVault et al
+    \thanks{Complete list available at \url{https://github.com/swaywm/wlroots/graphs/contributors}}}
+\maketitle
+
+\begin{abstract}
+wlroots is a flexible and modular library upon which Wayland compositors can be
```

Comment:
I like stylized lowercase wlroots, but I wish we would make an exception for capitalization at the beginning of a sentence.

Reply:
-1, you don't write IOS instead of iOS or EBay instead of eBay at the begining of a sentence. See https://english.stackexchange.com/questions/9063/how-do-you-capitalize-a-proper-noun-such-as-iphone

Reply:
It's not fair to use examples where there is a capitalization in beginning of the word. This is a practical matter where it's difficult to scan because you're not expecting the beginning of a sentence. Even Reddit now capitalizes their name in official documents.

Reply:
elementary is also using this "always lowercase policy" (see https://elementary.io/brand). It seems that Reddit capitalizes its brand in the middle of sentences (it doesn't use the lowercase form in the middle and the capitalized form at the begining, it sticks to one form per document).